controls/cis_ocp_1_4_0/section-1.yml

```diff
@@ -136,94 +136,117 @@ controls:
     controls:
     - id: 1.2.1
       title: Ensure that anonymous requests are authorized
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_anonymous_auth
       level: level_1
     - id: 1.2.2
       title: Ensure that the --basic-auth-file argument is not set
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_basic_auth
       level: level_1
     - id: 1.2.3
       title: Ensure that the --token-auth-file parameter is not set
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_token_auth
       level: level_1
     - id: 1.2.4
       title: Use https for kubelet connections
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_https_for_kubelet_conn
+        - api_server_openshift_https_serving_cert
+        - api_server_oauth_https_serving_cert
       level: level_1
     - id: 1.2.5
       title: Ensure that the kubelet uses certificates to authenticate
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_kubelet_client_cert
+        - api_server_kubelet_client_cert_pre_4_9
+        - api_server_kubelet_client_key
+        - api_server_kubelet_client_key_pre_4_9
       level: level_1
     - id: 1.2.6
       title: Verify that the kubelet certificate authority is set as appropriate
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_kubelet_certificate_authority
       level: level_1
     - id: 1.2.7
       title: Ensure that the --authorization-mode argument is not set to AlwaysAllow
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_auth_mode_no_aa
       level: level_1
     - id: 1.2.8
       title: Verify that RBAC is enabled
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_auth_mode_rbac
```

Comment:
The rule's control reference says 1.2.9. So either the rule was put into the wrong control or the rule's control reference needs to be fixed.